client/rest-high-level/src/test/java/org/elasticsearch/client/SecurityRequestConvertersTests.java

```diff
@@ -101,6 +104,25 @@ public void testPutRoleMapping() throws IOException {
         assertToXContentBody(putRoleMappingRequest, request.getEntity());
     }
 
+    public void testGetRoleMappings() throws IOException {
+        int noOfRoleMappingNames = randomIntBetween(0, 2);
+        final String[] roleMappingNames = randomArray(noOfRoleMappingNames, noOfRoleMappingNames, String[]::new, () -> randomAlphaOfLength(
```

Comment:
this might read better if you wrap after the `=` so the right hand side is all on one line